### PR TITLE
fix: keep unrecognized configuration when ensure lb configmap

### PR DESCRIPTION
**Pull request by hanxueluo**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @hicaistar @sinksmell 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/proxy/nginx/config.go [hunk 1]
```diff
@@ -38,16 +38,32 @@ var (
 		"server-tokens":          "false",
 		"skip-access-log-urls":   "/nginx_status/format/json",
 	}
+
```
Comment (hicaistar) [inline pkg/proxy/nginx/config.go]: 可不可以考虑将 defaultConfig 用 一个 configMap 保存，而且支持修改

Comment (hanxueluo) [reply pkg/proxy/nginx/config.go]: 经讨论，暂不采用这种方案。

**Pull request merged**